packages/apps/graph/src/services/account-service.ts

```diff
@@ -48,20 +48,13 @@ export async function getNonFungibleChainAccount({
   chainId: string;
   accountName: string;
 }): Promise<NonFungibleChainAccount | null> {
-  const [accountDetails, tokenDetails] = await Promise.all([
-    accountDetailsLoader.load({ accountName, chainId, fungibleName: 'coin' }),
-    tokenDetailsLoader.load({ accountName, chainId }),
-  ]);
+  const tokenDetails = await tokenDetailsLoader.load({ accountName, chainId });
 
-  return accountDetails !== null && tokenDetails !== null
+  return tokenDetails !== null && tokenDetails.length !== 0
```

Comment:
@alber70g If no tokens are found, it will return null. If there is a `create-account` like action for marmalade, I can check if there's a way to check (not from reconcile view)

Reply:
For now we can leave it. An empty account isn't that interesting anyway.